project/PekkoDependency.scala

```diff
@@ -45,9 +45,8 @@ object PekkoDependency {
             // (typically for the docs that require 2.6)
             if (defaultVersion.startsWith("1.0")) Artifact(determineLatestSnapshot("0.0.0"), true)
             else Artifact(defaultVersion)
-          case Some("default") => Artifact(defaultVersion)
-          case Some(other)     => Artifact(other, true)
-          case None            =>
+          case Some(other)            => Artifact(other, true)
+          case Some("default") | None =>
```

Comment:
Can we add a `NOTE` here stating that this should be reverted once a pekko release is made?

Reply:
It doesn't have to be changed later, these two are actually supposed to be the same.

Reply:
But actually the ordering of clauses was now wrong... Fixed.

Reply:
👍